cluster/src/dunit/scala/io/snappydata/cluster/StringAsVarcharDUnitTest.scala

```diff
@@ -0,0 +1,183 @@
+package io.snappydata.cluster
+
+/*
+ * Copyright (c) 2016 SnappyData, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you
+ * may not use this file except in compliance with the License. You
+ * may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License. See accompanying
+ * LICENSE file.
+ */
+
+import java.sql.{Connection, DriverManager}
+
+import io.snappydata.test.dunit.AvailablePortHelper
+
+import org.apache.spark.Logging
+import org.apache.spark.sql.{SaveMode, SnappyContext}
+
+/**
+ * Tests for verifying rendering of STRING happens as VARCHAR or CLOB,
+ * depending upon the conf property or query hint.
+ */
+class StringAsVarcharDUnitTest(val s: String)
+    extends ClusterManagerTestBase(s) with Logging {
+
+  val tableName1 = "colTab1"
+
+  val tableName2 = "colTab2"
+
+  override def tearDown2(): Unit = {
+    super.tearDown2()
+  }
+
+  private def getANetConnection(netPort: Int,
+      useGemXDURL: Boolean = false): Connection = {
+    val driver = "com.pivotal.gemfirexd.jdbc.ClientDriver"
```

Comment:
I think we should take this method in the base class and everyone should just use it. This code is repeated everywhere.